cookiecutter.json

```diff
@@ -3,5 +3,5 @@
     "formal_name": "App Name",
     "organization_name": "Example Corporation",
     "dir_name": "iOS",
-    "bundle": "com.example",
+    "bundle": "com.example"
```

Comment:
Is this a matter of JS convention, or is there a parsing issue re: trailing commas in JSON I'm not familiar with?